app/services/sms_sender.rb

```diff
@@ -24,6 +24,8 @@ def formatted_content(content)
 
   def perform
     case @provider.to_sym
+    when :sms_factor
+      send_with_smsfactor
```

Comment:
```suggestion
      send_with_sms_factor
```

pour la cohérence avec le reste